### ISSUE TITLE
Not syncing with gnome shell calendar

### ISSUE THREAD

**Issue by maierjeff** [alainm23/planify#139]
Planner recognize my calendars on my computer, at the same time i set-up. But, the activities aren't displayed on planner, as the planner activities aren't displayed on my shell. 

I'm a designer, I don't know anything about coding, and puttin it here because I don't find a forum. 
I use ubuntu 19.10, with xorg

**Comment by alainm23** [comment]
Could you run Planner with this command and send me the result of the terminal please.
`flatpak run --log-session-bus com.github.alainm23.planner `

**Comment by maierjeff** [comment]
[planner.txt](https://github.com/alainm23/planner/files/4087353/planner.txt)

**Comment by hwille** [comment]
do you have any feedback on this ? 
I'm having the same problem (ClearOS)
[plannerlog.txt](https://github.com/alainm23/planner/files/4281239/plannerlog.txt)

**Comment by romixch** [comment]
Is this an issue that only appears with flatpak? I can reproduce it when I start Planner with flatpak. When I start my own compiled version, I get all the calendar entries.
Wild speculation: It has probably to do with some flatpak permissions that need to be granted to Planner?

**Comment by Dirli** [comment]
Hello, From the description of the problem, it looks very much like you have the same problem as in the indicator. You are not subscribed to add-remove signals.

**Comment by Dirli** [comment]
Or rather you don't emit them

**Comment by alainm23** [comment]
@Dirli  Great, I'm going to try to rewrite my code based on your repository.
https://github.com/Dirli/wingpanel-indicator-datetime
Hope this can work. Thank you.

**Comment by Dirli** [comment]
No hurry. I will try to install Ubuntu tomorrow and understand exactly what the error is)

**Comment by alainm23** [comment]
Thank you very much, I think the error has something to do with `libecal-2.0` but I'm not sure too.
In my tests with Pop! OS I got this error:

`** (com.github.alainm23.planner:2): CRITICAL **: 14:54:34.251: CalendarModel.vala:208: Unable to connect to “Lembrete de aniversário & bodas”: The name org.gnome.evolution.dataserver.Calendar7 was not provided by any .service files`

Do you have any idea why it happens?
The service file is here:
https://github.com/alainm23/planner/blob/master/src/Services/Calendar/CalendarModel.vala

**Comment by Dirli** [comment]
Does not find the dbus-service. Check whether you have this dbus-service. I'm confused by the 7 at the end of the org.gnome.evolution.dataserver.Calendar7

**Comment by alainm23** [comment]
Yes, `org.gnome.evolution.dataserver.Calendar7` is the name of the d-bus server that comes by default in elementary OS Hera and that I'm requesting permissions here [flatpak](https://github.com/alainm23/planner/blob/master/build-aux/flatpak/com.github.alainm23.planner.json#L17).

I just did some testing with [D-Feet](https://wiki.gnome.org/Apps/DFeet) in Pop! OS 19.10 and they use `org.gnome.evolution.dataserver.Calendar8`

So I think the problem is the version of EDS that I am using:
https://github.com/alainm23/planner/blob/master/build-aux/flatpak/com.github.alainm23.planner.json#L73

![image](https://user-images.githubusercontent.com/33765137/80041347-daa4a800-84c1-11ea-9bba-01da916659b7.png)

**Comment by alainm23** [comment]
I think the problem is the version of `libedataserver` that Planner uses (`v3.28`) I'm trying to [build](https://github.com/alainm23/planner/blob/master/build-aux/flatpak/com.github.alainm23.planner.json) Planner with ``libedataserver` v3.36` (The same one used by [Gnome Calendar](https://github.com/flathub/org.gnome.Calendar/blob/master/org.gnome.Calendar.json#L100)) EDS builds correctly but building Planner gives me this error.:
```
error: Package `libedataserver-1.2` not found in specified Vala API directories or GObject-Introspection GIR directories
error: Package `libical-glib' not found in specified Vala API directories or GObject-Introspection GIR directories
error: Package `libecal-2.0' not found in specified Vala API directories or GObject-Introspection GIR directories
Compilation failed: 3 error(s), 0 warning(s)
```
Hi @tintou, please if you have time, could you give me any clue what is going wrong?

**Comment by tintou** [comment]
@alainm23 you need to enable the GObject Introspection and Vala bindings when building libical (
`-DGOBJECT_INTROSPECTION=true`, `-DICAL_GLIB_VAPI=true`, `-DICAL_GLIB=true`) and evolution data server (`-DENABLE_VALA_BINDINGS=ON`, `-DENABLE_INTROSPECTION=ON`)

**Comment by alainm23** [comment]
Calendar Events works 🎉️🎉️🎉️ Thank you very much @tintou , you are a genius.
![PlannerPopos](https://user-images.githubusercontent.com/33765137/83195145-6207bb80-a0ff-11ea-9fce-34909a3edd0b.png)

**Comment by jughn** [comment]
Can we fix that too? I haven't compiled the code myself, running off flathub now, but this is a key feature I'd love to have asap

**Comment by fischer-felix** [comment]
Hi, I just installed planner, and I am loving it so far, but unfortunately my calendar sync with gnome calendar isn't working either. I'm on Manjaro Linux, and I am using the latest AUR build of planner (2.6.9) and version 3.38.2 of gnome calendar on KDE plasma 5.20. I also tried installing it via flatpak, but nothing changed. Do I maybe have to install additional dependencies?
[plannerlog.txt](https://github.com/alainm23/planner/files/5991508/plannerlog.txt)
